src/helpers/analyticsEvents/studyGuides/openCloseColorKey.ts

```diff
@@ -0,0 +1,24 @@
+import { createSelector } from 'reselect';
+import * as selectNavigation from '../../../app/navigation/selectors';
+import { AnalyticsEvent } from '../event';
+
+const openText = 'REX Study guides (open color key)';
+const closeText = 'REX Study guides (close color key)';
+
+export const selector = createSelector(
+  selectNavigation.pathname,
+  (pathname) => ({pathname})
+);
+
+export const track = (
+  {pathname}: ReturnType<typeof selector>,
+  open: boolean
+): AnalyticsEvent | void => {
+  return {
+    getGoogleAnalyticsPayload: () => ({
+      eventAction: 'button',
```

Comment:
this action doesn't seem helpful here.

i'm thinking we can make the category `REX Study guides (color key)` and the action either `open` or `close`